### PR TITLE
Esteban biz fn

### PR DESCRIPTION
**Pull request by esteban-gs**

### CODE REVIEW

**Comment by esteban-gs** [comment]
Added maybe one line of code to config_inc.php:
`setlocale(LC_MONETARY,"en_US");`

**Comment by thomaskise** [comment]
did you make any changes to inc_0700/cinfig.php?

**Comment by thomaskise** [comment]
inc_0700/config.php?

**Comment by esteban-gs** [comment]
Yes. Line 327 is the only code I added in that file.

File: inc_0700/config_inc.php [hunk 1]
```diff
@@ -1,327 +1,330 @@
...
+        fwrite($isOpen,$myStr);
+        fclose($isOpen);
+        return TRUE;
+  }else{
+        return FALSE;
+  }
+}#End fileWrite()
+
+
+setlocale(LC_MONETARY,"en_US");
```
Comment (esteban-gs) [inline inc_0700/config_inc.php:327]: @thomaskise this line

**Comment by thomaskise** [comment]
got it. Thanks

File: inc_0700/config_inc.php (hunk 1 above)
Comment (thomaskise) [reply inc_0700/config_inc.php:327]: got it thanks!

**Pull request merged**